README.md

```diff
@@ -15,6 +15,7 @@ The examples folder contains example solutions across a variety of Google Cloud
 * [CloudML Sentiment Analysis](examples/cloudml-sentiment-analysis) - Sentiment analysis for movie reviews using TensorFlow `RNNEstimator`.
 * [CloudML TensorFlow Profiling](examples/tensorflow-profiling-examples) - TensorFlow profiling examples for training models with CloudML
 * [Dataflow BigQuery Transpose Example](examples/dataflow-bigquery-transpose) - An example pipeline to transpose/pivot/rotate a BigQuery table.
+* [Dataflow BigTable CyptoCurrencies Exchange RealTime Example](examples/cryptorealtime) - Apache Beam example that reads from the Crypto Exchanges WebSocket API and saves the feed in Cloud Bigtable. Real time visualization from BigTable running on Flask server in included.
```

Comment:
Please refer to these products as "Cloud Dataflow" and "Cloud Bigtable" everywhere, not simply "Dataflow" or "Bigtable".

I realize this has already been done for previous solutions, but if you could fix the newly-added lines, that would be great.